packages/web/src/scss/components/PricingPlan/README.md

```diff
@@ -0,0 +1,196 @@
+# PricingPlan
+
+The PricingPlan component provides a flexible and structured way to display
+pricing information and feature comparisons. It's ideal for subscription plans,
+service tiers, or product packages.
+
+PricingPlan is a composition of several subcomponents:
+
+- [PricingPlan](#pricingplan) – The main container
+  - [PricingPlanHeader](#pricingplanheader) – Contains plan title, price, and call-to-action
+  - [PricingPlanBody](#pricingplanbody) – Contains the feature list
+  - [PricingPlanFooter](#pricingplanfooter) – Optional additional information at the bottom
+
+## PricingPlan
+
+This is the main container of the composition.
+
+```html
+<article class="PricingPlan">
+  <div class="PricingPlan__layout">
+    <!-- PricingPlanHeader -->
+    <!-- PricingPlanBody -->
+    <!-- PricingPlanFooter -->
+  </div>
+</article>
+```
+
+ℹ️ By default, PricingPlan uses the `<article>` element at the root level.
+You can change it to any HTML element, just make sure the whole PricingPlan
+composition produces valid HTML output.
+
+ℹ️ There is an additional `<div>` with the `.PricingPlan__layout` class that
+establishes named grid lines to place PricingPlan content on. The additional
+element is necessary since in the most common scenario of PricingPlan usage, the
+grid lines of the `.PricingPlan` element are overridden by the [Matrix][matrix]
+component to [subgrid][mdn-subgrid], so any grid line names would be lost.
+
+## PricingPlanHeader
+
+The header contains the plan's title, optional subtitle, price, and
+a call-to-action button. It can also include an optional badge and note.
```

Comment:
Note is optional as well, or not? Maybe we could group mandatory and optional subcomponents.